sdc/tests/test_dataframe.py

```diff
@@ -1473,30 +1482,67 @@ def test_impl(df, df2):
         df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n)**2})
         df2 = pd.DataFrame({'A': np.arange(n), 'C': np.arange(n)**2})
         df2.A[n // 2:] = n
+        hpat_func(df, df2)
         pd.testing.assert_frame_equal(hpat_func(df, df2), test_impl(df, df2))
 
-    @unittest.skip("needs dict typing in Numba")
+    @skip_sdc_jit
+    def test_isin_df2(self):
+        def test_impl(df, df2):
+            return df.isin(df2)
+
+        cfunc = self.jit(test_impl)
```

Comment:
Please use consistent names across all the tests, e.g test_impl, hpat_func, etc.